client/src/Components/Header.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import IconButton from "@material-ui/core/IconButton";
+import Typography from "@material-ui/core/Typography";
+import SearchIcon from "@material-ui/icons/Search";
+import SettingsIcon from "@material-ui/icons/Settings";
+import TextField from "@material-ui/core/Input";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles((theme) => ({
+  typography: {
+    fontFamily: theme.typography.fontFamily,
+  },
+  palette: {
+    background: theme.palette.primary.main,
```

Comment:
Same for this. It will already use the palette for the theme primary color by default